CHANGELOG.md

```diff
@@ -10,7 +10,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-Changed OkHttpRequestAdapter dependency from OkHttpClient to Call.Factory (parent interface implemented by OkHttpClient).
+## [1.0.0] - 2023-11-10
```

Comment:
can we use a minor version for the time being? And want to make sure things are aligned before we major rev.